packages/cryptoplease/lib/core/accounts/bl/accounts_state.dart

```diff
@@ -4,6 +4,14 @@ part of 'accounts_bloc.dart';
 class AccountsState with _$AccountsState {
   const factory AccountsState({
     MyAccount? account,
+    AccessMode? accessMode,
```

Comment:
I don't like that `accessMode` and `account` are "independently" nullable. Let's maybe make `accessMode` part of `MyAccount`, as they are tightly coupled.